django/db/models/aggregates.py

```diff
@@ -166,14 +167,16 @@ class Count(Aggregate):
     output_field = IntegerField()
     allow_distinct = True
     empty_result_set_value = 0
+    arity = 1
     allows_composite_expressions = True
 
-    def __init__(self, expression, filter=None, **extra):
```

Comment:
Looking again, I guess there is a subtle incompatibility here if someone passed `filter` by positional arg. Allow me a second to check/think on it?

Reply:
I think for these functions where I changed `expression` to `*expressions` I was trying to handle a problem that doesn't exist. Multiple arguments already either raised a TypeError from python. I'll rework this a little bit.

Reply:
Reworked this with a lighter touch.